mozfun/event_analysis/create_events_view/README.md

```diff
@@ -0,0 +1,181 @@
+### Usage
+
+```
+create_events_view(
+    view_name STRING,
+    project STRING,
+    dataset STRING,
+    funnels ARRAY<STRUCT<
+        funnel_name STRING,
+        funnel ARRAY<STRUCT<
+            step_name STRING,
+            events ARRAY<STRUCT<
+                category STRING,
+                event_name STRING>>>>>>,
+    counts ARRAY<STRUCT<
+        count_name STRING,
+        events ARRAY<STRUCT<
+            category STRING,
+            event_name STRING>>>>
+  )
+```
+
+***view_name***: The name of the view that will be created. This view
+    will be in the shared-prod project, in the analysis bucket,
+    and so will be queryable at `\`moz-fx-data-shared-prod\`.analysis.{view_name}`.
+
+***project***: The project where the `dataset` is located.
+
+***dataset***: The dataset that must contain both the `events_daily` and
+    `event_types` tables.
+
+***funnels***: An array of funnels that will be created. Each funnel has
+    two parts:
+1. `funnel_name`: The name of the funnel is what the column representing
+    the funnel will be named in the view. For example, with the value
+    `"onboarding"`, the view can be selected as follows:
+    ```
+    SELECT onboarding
+    FROM `moz-fx-data-shared-prod`.analysis.{view_name}
+    ```
+2. `funnel`: The ordered series of steps that make up a funnel.
+    Each step also has:
+    1. `step_name`: Used to name the column
+        within the funnel and represents whether the user completed
+        that step on that day. For example, within `onboarding` we may
+        have `completed_first_card` as a step; this can be queried at
+        ```
+        SELECT onboarding.completed_first_step
+        FROM `moz-fx-data-shared-prod`.analysis.{view_name}
+        ```
+    2. `events`: The set of events which indicate the user completed
+        that step of the funnel. Most of the time this is a single event.
+        Each event has a `category` and `event_name`.
+
+***counts***: An array of counts. Each count has two parts, similar to funnel steps:
+1. `count_name`: Used to name the column representing the event count. E.g.
+    `"clicked_settings_count"` would be queried at
+    ```
+    SELECT clicked_settings_count
+    FROM `moz-fx-data-shared-prod`.analysis.{view_name}
+    ```
+2. `events`: The set of events that we want to count. Each event has
+    a `category` and `event_name`.
+
+#### Structure of the Resulting View
+
+The view will be located at `\`moz-fx-data-shared-prod`\.analysis.{event_name}`.
+
```

Comment:
I think it would be good to provide some kind of table or simple mermaid diagram illustrating what the generated BigQuery table would look like.

Reply:
Honestly it's not clear to me what that would even look like. Can you be more specific on what you might want to see?

Reply:
How about a simple schema diagram like this?

https://docs.telemetry.mozilla.org/datasets/search/search_aggregates/reference.html#schema

Reply:
@wlach Added one, let me know if it's reasonable. I didn't want to make it too long so I left out the dimensions.